README.md

```diff
@@ -246,6 +246,28 @@ _Sample apps using various components._
 
 * [**Glean**](samples/glean) - An app demoing how to use the [**Glean**](components/service/glean/README.md) library to collect and send telemetry data.
 
+# Building #
+
+## Command line ##
+
+```
+$ git clone https://github.com/mozilla-mobile/android-components.git
+$ cd android-components
+$ ./gradlew
+```
+
+## Android Studio ##
+
+If the environment variable `JAVA_HOME` is not defined, you will need to set it. If you would like to use the JDK installed by Android Studio, here's how to find it: 
+
+1. Open Android Studio.
+2. Select "Configure".
+3. Select "Default Project Structure". You should now see the Android JDK location. 
+4. Set the environment variable `JAVA_HOME` to the location. (How you set an environment variable depends on your OS.)
+5. Restart Android Studio.
+
+Once the environment variable is set, you can import the project into Android Studio with the default wizard options.
```

Comment:
Should we including running gradle sync or any other steps after opening the project?

Reply:
That should be done automatically after the first clone of a-c and when we try to build.